neural_cherche/retrieve/sparse_embed.py

```diff
@@ -330,8 +330,10 @@ def _retrieve(
         k
             Number of documents to retrieve.
         """
-        similarities = -1 * embeddings["sparse_activations"].dot(self.matrix)
-        sparse_matchs, _ = self.top_k(similarities=similarities, k=k)
+        #similarities = -1 * embeddings["sparse_activations"].dot(self.matrix)
+        #sparse_matchs, _ = self.top_k(similarities=similarities, k=k)
+        sparse_matchs=[ i.indices for i in  (embeddings["sparse_activations"].dot(self.matrix))]
```

Comment:
The contextual scoring function of SparseEmbed is quite costly, do you think it's better to not filter that match the sparse scoring function in order to rerank top_k documents?

Reply:
Hi @arthur-75, kudos for the update of the pull-request, you did not respond to this specific comment, maybe because you did not send your comments ? You can use "add single comment" button in order to send your feedback otherwise you can use "start a review" and then press another button to validate the review. 

<img width="645" alt="image" src="https://github.com/raphaelsty/neural-cherche/assets/24591024/308cc485-81c9-4269-8963-a733103d2e8b">